src/_guides/language/sound-problems.md

```diff
@@ -20,7 +20,27 @@ and see [these other resources](/guides/language/type-system#other-resources).
 
 ## Troubleshooting
 
-### No type errors {#no-type-errors}
+Dart enforces a sound type system. Roughly, this means
+you can't get into a situation where the value stored in a variable is
+different from the variable's static type. Like most modern statically
+typed languages, Dart accomplishes this with a combination of
+[static (compile-time)](#static-errors-and-warnings) and [dynamic (runtime)](#runtime-errors) checking.
+
+For example, the following type error is detected at compile-time:
+
+{:.fails-sa}
+{% prettify dart tag=pre+code %}
+List<int> numbers = [1, 2, 3];
+List<String> [!string = numbers!];
+{% endprettify %}
+
+Since neither `List<int>` nor `List<String>` is a subtype of the other,
+Dart rules this out statically. 
+
+You can see other examples of static analysis errors, as well as other error types, in the sections below.
```

Comment:
```suggestion
You can see other examples of static analysis errors,
as well as other error types, in the following sections.
```

It's a holdover from the old days, but I do try to avoid "above" and "below" as, if this is translated to Chinese or whatever, that isn't relevant.